app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -524,6 +526,7 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener {
         toggle_photo_video.setImageResource(R.drawable.ic_camera)
         countdown_toggle.beInvisible()
         countdown_times.beInvisible()
+        filter_scroll_area.beInvisible()
```

Comment:
although this will make it so that when you enter video mode the filters disappear, you can still just swipe up to toggle them visible again. Add logic in `toggleFilterScrollArea()` function to make visible only if in camera mode

Reply:
Good eye, I'll make the change